packages/communication-react/CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ Fri, 09 Jul 2021 20:41:33 GMT
 
 ### Changes
 
+* Updated calling sdk dependency to 1.2.0-beta.1
+
+* UI Update improvements for Calling UI Components
+
+* Updates to selectors and adapters. Learn more about these updates in aka.ms/acsstorybook
```

Comment:
Maybe update this to be "Read our updated documentation at aka.ms/acsstorybook" - they can't really learn more about the updates themselves here

Reply:
This was related to selectors and adapters but I can remove this part